pitch.md

```diff
@@ -23,38 +58,137 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
 by describing which unicorn properties (horn color, tattoo, etc.) are most 
 important to users and what kinds of values those properties might have._ 
 
+I don't know my exact syntax. But for the sake of simplicity, let us focus on 
+"full circle" moves that do not touch the body, shorten string length, or 
+involve string manipulation.
+
+There are two main properties that distinguish one move from another:
+
+1. Where the glowsticks are in relation to the body
+	* Plane
+		1. Wall - perpendicular to front or back of body
+		2. Wheel - perpendicular to left or right of body
+		3. Floor - perpendicular to the floor
+	* Is the arm rotating (moving in circles), and if so, counterclockwise or clockwise?
+	* Is the glowstick moving counterclockwise or clockwise relative to the hand?
+	* Start and end position of the hand after one full 360° circle.
+2. Where the glowsticks are in relation to each other
+	* This can be represented as offset angle phases, as many moves have both glowsticks doing the same motion but starting at different phase positions.
+
+Ideally, with this information, we can build individual tricks based on these 
+properties for a "full circle".
+
 
 ## Operation (semantics)
 _What might happen when a program runs? Is there a computational model that
 corresponds with your domain? How does a program interact with the
 user? What kinds of semantic (i.e., non-syntax) errors might occur, and how 
 might they be communicated to the user?_
 
+When a program runs, I want to create tricks based on the properties of 
+glowsticking motions. Perhaps this can be represented as a time sequence of 360° 
+motions with different properties (see syntax). In the far future, the results 
+of my DSL can be translated into an API (internal representation) that can be 
+used with poi simulators (see related work). 
+
+I think data interpretation is one possible computational model because we want 
+to describe our input (as glowsticking tricks). This would be displayed through 
+a user interface like a 2D/3D simulator.
+
+Some errors might be the user not providing enough info for a single motion. 
```

Comment:
Perhaps the user may also define a set of moves that would actually be impossible to form in real life? I'm not completely sure how you would identify this, but it seems like a reasonable error that might occur.